### DISCUSSION TITLE
Too Many Community Emails? - Update Your Community Settings or Mute the Thread

### DISCUSSION THREAD

**Discussion by bahenry** [SalesforceCommerceCloud/sfcc-ci#523] [Team Posts]
Everyone who has joined the Salesforce Commerce Cloud Community has likely seen discussion emails come across their mailboxes. You can reduce the amount of emails by muting a thread with the link in the bottom of an email -or- update your notification settings from _Watching_ to _Not Watching_.

MUTE THREAD
<img width="419" alt="mute-the-thread" src="https://user-images.githubusercontent.com/2481883/53851332-07c38d80-3f84-11e9-8ded-a396525199ba.png">

VISIT COMMUNITY IN GITHUB TO UNWATCH
<img width="384" alt="notification settings" src="https://user-images.githubusercontent.com/2481883/53851348-0eea9b80-3f84-11e9-97ee-e0979b493270.png">

**Comment by bahenry** [comment]
Or _Unsubscribe from the discussion_ in Github

<img width="810" alt="unsubscribe" src="https://user-images.githubusercontent.com/2481883/53851651-f0d16b00-3f84-11e9-9c15-2db7d79fd830.png">

**Comment by z1haze** [comment]
Or stop using github as a forum :)

**Comment by bahenry** [comment]
> Or stop using github as a forum :)

Absolutely. Note that the Community topic specifically mentions this. 

"Commerce Cloud Community. While you can post discussions in this group, keep in mind there are better alternatives, such as xchange.demandware.com, developer.salesforce.com, partners.salesforce.com"

And yes, there is Slack too https://sfcc-unofficial.slack.com/ which is great for casual dialogue, but does not ultimately contribute to the long term knowledge growth of the Community as does the more lasting forums mentioned above.
